packages/components/src/List/ListComposition/ColumnChooser/ColumnChooserHeaderButton.component.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { ButtonIconFloating } from '@talend/design-system';
+
+import ColumnChooser from './ColumnChooser.component';
+import ColumnChooserButton from '../../Toolbar/ColumnChooserButton';
+import theme from './ColumnChooserHeaderButton.scss';
+
+export default function ColumnChooserHeaderButton(props) {
+	return (
+		<div className={theme.columnChooserHeaderButtonContainer}>
```

Comment:
we usually use kebab case for class names

```suggestion
		<div className={theme['column-chooser-header-button-container']}>
```